utils/testrunner/README.md

```diff
@@ -11,7 +11,7 @@ This test runner is used internally by Playwright to test Playwright itself.
 ### Installation
 
 ```sh
-npm install --save-dev @pptr/testrunner
+npm install --save-dev @playwright/testrunner
```

Comment:
we can remove the installation part as well

Reply:
Done.